receiver/snmpreceiver/integration_test.go

```diff
@@ -64,7 +64,7 @@ func TestSnmpReceiverIntegration(t *testing.T) {
 
 	for _, testCase := range testCases {
 		t.Run(testCase.desc, func(t *testing.T) {
-
+			t.Skip("Flaky test, see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21086")
```

Comment:
nit - I think we could put this at the top level of the test function so it doesn't call `Skip` on every test case here.